python/ray/_private/worker.py

```diff
@@ -541,6 +543,16 @@ def runtime_env(self):
         """Get the runtime env in json format"""
         return self.core_worker.get_current_runtime_env()
 
+    @property
+    def debugger_port(self):
+        """Get the debugger port for this worker"""
+        return self._debugger_port
+
+    def set_debugger_port(self, port):
+        worker_id = self.core_worker.get_worker_id()
```

Comment:
nit: I wonder if we just want to embed the `update_worker_debugger_port` implementation here in the `worker.py` file. 

E.g. through `ray._private.state.state.global_state_accessror` like:

https://github.com/ray-project/ray/blob/70dbcbffddf0bf05f48d27f89f981951672cee34/python/ray/_private/worker.py#L2627-L2635

Reply:
I will defer this to you and others in the core team to give the direction, I don't think we want this to be an external API and I do see few usage `ray._private.state`, so not sure what the difference here between the two.